container/sagemaker/tensorflow-serving.js

```diff
@@ -120,6 +120,19 @@ function parse_custom_attributes(r) {
             }
         }
     }
+
+    // for dynamic endpoint invocations, tfs-model-name is in the uri, or use default_tfs_model
```

Comment:
this needs to happen before versions, otherwise the uri will not be valid TFS uri